setup.cfg

```diff
@@ -39,7 +39,7 @@ install_requires =
     attrs>=21.4.0
     frozendict
     dulwich>=0.20
-    py_randomprime>=1.13.0
+    py_randomprime>=1.14.1
```

Comment:
~~what happened to 1.14.2~~

Reply:
The CI uses the oldest tag if there are duplicates I guess. The pypi package is .1